test/plugin/test_out_numeric_monitor.rb

```diff
@@ -404,4 +404,34 @@ def test_output_key_prefix
     assert_equal 12, r['key_prefix_sum']
     assert_equal 6, r['key_prefix_num']
   end
+
+  def test_emit_with_nil_values
+    d = create_driver(CONFIG)
+    begin
+      d.run(default_tag: 'test.tag1') do
+        10.times do
+          d.feed({'field1' => nil})
```

Comment:
Does this test need 10 input records?
If just 1 input record is enough to cause the error, use just 1 record.